### ISSUE TITLE
How to Focus on local part

### ISSUE THREAD

**Issue by bookkkker** [lllyasviel/ControlNet#684]
Now I need to generate a defect on the industrial photo, how to revise the loss function or any other methods to help model learns the useful part of my training data.
